src/Symfony/Component/Console/Output/OutputInterface.php

```diff
@@ -48,6 +48,11 @@ public function write($messages, $newline = false, $options = 0);
     public function writeln($messages, $options = 0);
 
     /**
+     * Clears the screen of output.
+     */
+    public function clear();
```

Comment:
Adding new methods in the interface is a BC break (btw, you haven not even updated all implementations in the component, so the component itself is affected by the BC break)